sql/common/src/test/scala/org/apache/sedona/sql/rasteralgebraTest.scala

```diff
@@ -403,6 +403,20 @@ class rasteralgebraTest extends TestBaseScala with BeforeAndAfter with GivenWhen
       assertEquals(rasterDf.count(), binaryDf.count())
     }
 
+    it("Passed RS_UpperLeftX"){
+      val df = sparkSession.read.format("binaryFile").load(resourceFolder + "raster/test1.tiff")
+      val result = df.selectExpr("RS_UpperLeftX(RS_FromGeoTiff(content))").first().getDouble(0)
+      val expected: Double = -1.3095817809482181E7
+      assertEquals(expected, result, 1e12)
```

Comment:
This will allow a difference of 10^12 while asserting equals. I think you mean 1e-12 which is 10^-12

Reply:
Thank you for pointing it out.